js/setup.js

```diff
@@ -3,23 +3,36 @@ $(function() {
 
    function createPageTable() {
       var $pageTable = $('<div class="qa-tbl"></div>');
-      var tableHeader = '<div class="qa-tbl-row qa-tbl-hdr qa-stripe"><div class="qa-page-id qa-center"><span>Page ID</span></div>' +
-          '<div class="qa-page-title"><span>Page Title</span></div><div class="qa-page-q-count qa-center"><span>Question Count</span>' +
-          '</div><div class="qa-page-preview qa-center"><span>Preview</span></div></div>';
+      var $templates = $('.qa-templates');
+      var tableHeader = $templates.find('.js-page-hdr')
+          .clone()
+          .children();
 
       $pageTable.append(tableHeader);
 
       for(var i = 0; i < pages.length; i++) {
          var page = pages[i];
```

Comment:
Try to extract a function from this code. It'll help in maintaining the project.